build-system/tasks/presubmit-checks.js

```diff
@@ -954,8 +954,7 @@ const forbiddenTermsSrcInclusive = {
   '\\.scrollingElement(?!_)': bannedTermsHelpString,
   '\\.computeCTM(?!_)': bannedTermsHelpString,
   // Functions
-  '\\.changeHeight\\(': bannedTermsHelpString,
-  '\\.changeSize\\(': bannedTermsHelpString,
+  '\\.applySize\\(': bannedTermsHelpString,
```

Comment:
To clarify: `forceChangeHeight` and `forceChangeSize` are not included here because their new naming suggests the severity without explicitly needing to ban and `/* OK */` usage?

Reply:
Yep! That was the idea.